recipe/meta.yaml

```diff
@@ -11,38 +11,40 @@ source:
   sha256: 5cac870d6d269cdd71d28def4cae310c609c02aac636c0831176188f16021f44
 
 build:
-  number: 0
-  noarch: python
+  skip: True  # [py<37]
+  number: 1
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
+  build:
+    - {{ compiler('c') }}
+    - numpy
```

Comment:
this should not be in the build section.

Reply:
if you want to have a lower bound on numpy, you can add numpy "twice" to the host section, but conda-forge's numpy pinning is at 1.21 which is greater than 1.14 so I would simply omit the lower bound on numpy.

Otherwise, yu are liklely building with numpy 1.24 which will make your package incompatible with numpy 1.21.

Reply:
@thewchan let's discuss here on Mark's comment. I do not see any instruction to remove the compiler section. Only the numpy dependency. The thing is, does this package requires numpy headers?  It does looks like it based on https://github.com/predict-idlab/plotly-resampler/blob/b1225bd05d5d93821e06cb30fa589759e768a40b/pyproject.toml#L104

That means we need a `{{ pin_compatible('numpy') }}` in the `run` section and only `numpy` in the host section. The min numpy can be enforced but it is not necessary b/c our min numpy is already >=1.14 (I think we are at >=1.20 now).

Reply:
also see https://github.com/conda-forge/plotly-resampler-feedstock/pull/34